templates/documents/general_view.html

```diff
@@ -132,8 +142,10 @@ <h4>{$file->get_url_web()|text}
                 <button class="btn btn-xs btn-primary active">{xlt t='Contents'}</button>
               </div>
             <span style="float:right;">
-            <a class="css_button" href='' onclick='return popoutcontent(this)' title="{xla t='Pop Out Full Screen.'}">
-                <span class="glyphicon glyphicon-fullscreen"></span></a>
+            {if $GLOBALS.oefax_enable}
+            <a class="css_button btn btn-send-msg" href="" onclick="return doFax(event,{$file->get_url()|attr_js})"><span>{xlt t='Send Fax'}</span></a>
```

Comment:
And the use of `attr_js` makes the crowd go crazy :)
![crowd](https://user-images.githubusercontent.com/278968/64595024-fa89c000-d365-11e9-9145-2beee04adbe4.gif)